packages/snap/src/components/start-page.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  ZkCertStandard,
+  type ZkCertRegistered,
+} from '@galactica-net/galactica-types';
+import {
+  Banner,
+  Box,
+  Button,
+  Field,
+  FileInput,
+  Form,
+  Heading,
+  Text as SnapText,
+  type SnapComponent,
+} from '@metamask/snaps-sdk/jsx';
+
+import { CertsSection } from './certs-section';
+
+type Props = {
+  error?: string;
+  activeTab: ZkCertStandard;
+  holders: string[];
+  zkCerts: ZkCertRegistered[];
+};
+
+export const StartPage: SnapComponent<Props> = ({
+  error,
+  activeTab,
+  holders,
+  zkCerts,
+}) => {
+  const certs = zkCerts.filter((cert) => {
+    if (activeTab === ZkCertStandard.ZkKYC) {
+      return cert.zkCertStandard === ZkCertStandard.ZkKYC;
+    }
+    if (activeTab === ZkCertStandard.Twitter) {
+      return cert.zkCertStandard === ZkCertStandard.Twitter;
+    }
+    if (
+      activeTab === ZkCertStandard.ArbitraryData ||
+      ZkCertStandard.Exchange ||
+      ZkCertStandard.Rey
+    ) {
+      return (
+        cert.zkCertStandard !== ZkCertStandard.ZkKYC &&
+        cert.zkCertStandard !== ZkCertStandard.Twitter
+      );
+    }
+
+    return true;
+  });
+
+  console.log(certs);
```

Comment:
Please remove this log before merging because it could leak sensitive data to the browser log.